File: Python/J4/j4b.py
guthaben = 2000
zielguthaben = 2 * guthaben
zinssatz = 1.03
i = 0

while guthaben < zielguthaben:
    i += 1
    guthaben *= zinssatz
    if i == 1:
        print(f"Guthaben nach {i} Jahr: {round(guthaben, 2)} €")
    elif i > 1:
        print(f"Guthaben nach {i} Jahren: {round(guthaben, 2)} €")

print(f"\nnach {i} Jahren hat sich dein Guthaben verdoppelt!")

File: Java/Java-main/J4/j4b.java
public class j4b {
    public static void main(String[] args){
        double guthaben = 2000;
        double zielguthaben = 2 * guthaben;
        double zinssatz = 1.03;
        int i = 1;
        while(guthaben < zielguthaben){
            i++;
            guthaben = guthaben * zinssatz;
            System.out.println("Nach " + i + " Jahren " + " erreichst du den Betrag: " + guthaben + " euro ");
        }
    }
}